src/main/java/net/imglib2/view/Views.java

```diff
@@ -1356,6 +1375,24 @@ public static boolean isZeroMin( final Interval interval )
 		return interval( extendValue( source, t ), Intervals.expand( source, border ) );
 	}
 
+	/**
+	 * Expand a RandomAccessibleInterval as specified by border. source will be
+	 * extended with a constant value specified by the caller.
+	 *
+	 * @param source
+	 *            the interval to expand.
+	 * @param t
+	 *            Constant extension of source.
+	 * @return Expansion of the {@link RandomAccessibleInterval} source as
+	 *         specified by t and border.
+	 * @deprecated use {@code expandValue} with unbounded type parameter T
+	 */
+	@Deprecated
+	public static < T extends RealType< T > > IntervalView< T > expandValue( final RandomAccessibleInterval< T > source, final T t, final long... border )
```

Comment:
This can be relaxed to `Type<T>` instead of `RealType<T>`, no?

Reply:
Yes, it should be as it was before #268

Reply:
Oops, you're right, I guess..

Reply:
Now changed and force-pushed.